dist/build.js

```diff
@@ -2939,7 +2939,7 @@ function _setBranchViewData(a, b, c) {
 }
 Branch.prototype.setBranchViewData = wrap(callback_params.CALLBACK_ERR, function(a, b) {
   _setBranchViewData.call(null, this, a, b);
-});
+}, !0);
```

Comment:
I was so thrown off by the `!0` then realized this was the compilation. I forget what we did last time, commit build.js or not, either way, I know this isn't you directly.

Reply:
I guess the benefit of having this in the repo is that people who want to test with the local example.html don't have to install Clojure, Java, Python, compile, etc. It is generated by CD when deployed to prod and pushed back to the repo anyway. But I guess I'm not a fan of ever having to work from a local repo that isn't clean. You shouldn't have to avoid committing local changes that accompany a code change. I think I might prefer to remove these from the repo. If anyone wants to test with the local files, they can build. But we ought to have instructions for them in the repo.